index.html

```diff
@@ -36,5 +36,374 @@ <h1 id="head1">Hey there.<br> I’m Erick.<br>
            <img id="img" src="./features/images/Disabled.png">
       </section>
      </main>
+     <section>
+        <div id="content2"><p class="works">My Recent Works</p>
+           <img id="img2" src="./features/images/Indicator.png">
+        </div>
+         <div class="main-container">
+            <div class="grid-container">
+              <div class="item1"></div>
+               <div class="flex">
+                <div class="item2">Multi-Post Stories Gain+Glory</div>
+                  <ul class="flexbox">
+                    <li class="item3">Ruby on Rails</li>
+                    <li class="item3">CSS</li>
+                    <li class="item3">JavaScript</li>
+                    <li class="item3">HTML</li>
+                  </ul>
+                  <a href="#" target="_blank" class="item7">See project</a>
+                </div>
+              </div>
+              <div class="grid-container">
+                <div class="item1"></div>
+                 <div class="flex">
+                  <div class="item2">Multi-Post Stories Gain+Glory</div>
+                  <ul class="flexbox">
+                   <li class="item3">Ruby on Rails</li>
+                   <li class="item3">CSS</li>
+                   <li class="item3">JavaScript</li>
+                   <li class="item3">HTML</li>
+                 </ul>
+                  <a href="#" target="_blank" class="item7">See project</a>
+                 </div>
+               </div>
+               <div class="grid-container">
+                 <div class="item1"></div>
+                  <div class="flex">
+                   <div class="item2">Multi-Post Stories Gain+Glory</div>
```

Comment:
- [ ] Kindly consider using a header tag `hx` eg `h2, h3, h4, h5, h6` for the title of each card as it's semantically correct and also helps make your website accessible and SEO friendly.